packages/openneuro-app/src/scripts/refactor_2021/redesign-form.jsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react'
+import axios from 'axios'
+import queryString from 'query-string'
+import { Modal } from '@openneuro/components/modal'
+
+class ContactForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      modalities: [],
+      formCompleted: false,
+      showFeedbackModal: false,
+    }
+  }
+  myRequest = url => {
+    let response
+    try {
+      response = axios.post(url, null, null)
```

Comment:
We shouldn't add axios for one POST call. This can be easily done with fetch.
```suggestion
      response = fetch(url, {method: 'POST'})
```